tests/e2e/models/onboarding/BenefitsSection.ts

```diff
@@ -0,0 +1,23 @@
+import { Page } from 'playwright';
+
+export class BenefitsSection {
+	page: Page;
+
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	async isDisplayed() {
+		await this.page.waitForSelector(
+			'.woocommerce-profile-wizard__container.benefits'
+		);
+	}
+
+	async noThanks() {
+		// Wait for "No thanks" button to become active
+		await this.page.waitForSelector( ':text("No thanks")' );
```

Comment:
This is something that playwright solves for us and we should not need to use this very often (except when we just want to assert that something was displayed. like the above `isDisplayed()` method seems fine.

but things like `click` and `fill` wait for the selector to be present and so you don't have to do a wait between operating on elements.